ovirt-ansible-collection.spec.in

```diff
@@ -13,30 +13,19 @@ Url:		http://www.ovirt.org
 
 BuildRequires:	ansible-core
 BuildRequires:	ansible-test
-%if 0%{?rhel} > 7 && 0%{?rhel} < 9
-BuildRequires:	glibc-langpack-en
-%endif
 
 Requires:	ansible-core >= 2.13.0
 Requires:	ansible-collection-ansible-netcommon
 Requires:	ansible-collection-ansible-posix
 Requires:	ansible-collection-ansible-utils
 Requires:	qemu-img
-Requires:   python3-jmespath
+Requires:	python3-jmespath
 
-%if 0%{?rhel} >= 9
 Requires:	python3.11-ovirt-imageio-client
 Requires:	python3.11-ovirt-engine-sdk4 >= 4.5.0
 Requires:	python3.11-jmespath
 Requires:	python3.11-passlib
```

Comment:
Is the ansible still using 3.11 in el9?
I just checked and there might be problems with the python paths.

```
[root@18b9ee12e8e6 /]# ansible --version
ansible [core 2.14.18]
  config file = /etc/ansible/ansible.cfg
  configured module search path = ['/root/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3.9/site-packages/ansible
  ansible collection location = /root/.ansible/collections:/usr/share/ansible/collections
  executable location = /usr/bin/ansible
  python version = 3.9.21 (main, Dec  4 2024, 00:00:00) [GCC 11.5.0 20240719 (Red Hat 11.5.0-2)] (/usr/bin/python3)
  jinja version = 3.1.2
  libyaml = True
[root@18b9ee12e8e6 /]# python3 --version
Python 3.9.21

```

Reply:
It's not anymore indeed.

```
2023-07-17 - Dimitri Savineau <dsavinea@redhat.com> - 1:2.14.7-1
- ansible-core 2.14.7 release (rhbz#2221820)
- rebuild with python 3.9 (rhbz#2221820)
- remove bundled packaging, pyparsing and resolvelib.
```

So we can use the python3- variants again.
Thanks for that hint :) Good to see people still review patches ;)

Reply:
yeah, but sadly don't remember the sate of the other component if it has the default python builds or only the specific.
Sadly been busy with other work so not sure how active I can be

Reply:
https://github.com/oVirt/python-ovirt-engine-sdk4/commit/d75004346f4e937cb1bcb8bcd6884b2c7bff3b1e
https://github.com/oVirt/ovirt-imageio/commit/55b7eba0852a67e871e6a72436dbc61f5bbda5de

Seems that will need a fix, it's on my todo :)